code/modules/admin/admin.dm

```diff
@@ -929,6 +929,20 @@ var/global/nologevent = 0
 	if(!ai_number)
 		usr << "<b>No AIs located</b>" //Just so you know the thing is actually working and not just ignoring you.
 
+/datum/admins/proc/show_skills(var/mob/living/carbon/human/M as mob in world)
+	set category = "Admin"
+	set name = "Show Skills"
+
+	if (!istype(src,/datum/admins))

```

Comment:
> `if(0)`

Sure, if you're calling this on an admin datum. But this is used as a verb, attached to a client, in which case src will be... either the client or the client's mob? I'm not sure. But it *won't* be an admin datum, because it isn't attached to that in any way. This swaps it over to the client's admin datum (if it exists) in such a case.